build.gradle.kts

```diff
@@ -0,0 +1,230 @@
+    }
+    resolution.forceConfiguration(configurations)
+}
+
+// Apply some plugins to make type-safe extension accessors available in this script file.
+plugins {
+    java
+    idea
+    @Suppress("RemoveRedundantQualifierName") // Cannot use imports here.
+    id("com.google.protobuf").version(io.spine.gradle.internal.Deps.versions.protobufPlugin)
+}
+
+apply(from = "$rootDir/version.gradle.kts")
+
+extra.apply {
+    this["groupId"] = "io.spine"
+    this["publishToRepository"] = PublishingRepos.cloudRepo
+    this["projectsToPublish"] = listOf(
+            "base",
+            "tool-base",
+            "testlib",
+            "mute-logging",
+            "errorprone-checks",
+
+            // Gradle plugins
+            "plugin-base",
+            "javadoc-filter",
+            "javadoc-prettifier",
+            "proto-dart-plugin",
+            "proto-js-plugin",
+            "model-compiler",
+
+            "plugin-testlib",
+
+            // Protoc compiler plugin
+            "protoc-api",
+            "validation-generator",
+            "protoc-plugin"
+    )
+}
+
+allprojects {
+    apply {
+        plugin("jacoco")
+        plugin("idea")
+        plugin("project-report")
+        from("$rootDir/config/gradle/dependencies.gradle")
+    }
+    version = rootProject.extra["versionToPublish"]!!
+    version = rootProject.extra["versionToPublish"]!!
+}
+
+subprojects {
+    buildscript {
+        apply(from = "$rootDir/version.gradle.kts")
+
+        DependencyResolution.defaultRepositories(repositories)
+        dependencies {
+            classpath(Deps.build.gradlePlugins.protobuf)
+            classpath(Deps.build.gradlePlugins.errorProne)
+        }
+        DependencyResolution.forceConfiguration(configurations)
+    }
+
+    apply(from = "$rootDir/version.gradle.kts")
+
+    val sourcesRootDir by extra("$projectDir/src")
+    val generatedRootDir by extra("$projectDir/generated")
+    val generatedJavaDir by extra("$generatedRootDir/main/java")
+    val generatedTestJavaDir by extra("$generatedRootDir/test/java")
+    val generatedSpineDir by extra("$generatedRootDir/main/spine")
+    val generatedTestSpineDir by extra("$generatedRootDir/test/spine")
+
+    apply {
+        plugin("java-library")
+        plugin("pmd")
+        plugin("com.google.protobuf")
+        plugin("net.ltgt.errorprone")
+        plugin("maven-publish")
+        from(Deps.scripts.projectLicenseReport(project))
+    }
+
+    the<JavaPluginExtension>().apply {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    DependencyResolution.defaultRepositories(repositories)
+    dependencies {
+        "errorprone"(Deps.build.errorProneCore)
+        "errorproneJavac"(Deps.build.errorProneJavac)
+        Deps.build.protobuf.forEach { "api"(it) }
+        "api"(Deps.build.flogger)
+        "implementation"(Deps.build.guava)
+        "implementation"(Deps.build.checkerAnnotations)
```

Comment:
Are there calls that allow to say `implementation` only once, accepting a list of dependencies?